react-app/src/components/AdventureDetail.js

```diff
@@ -141,11 +95,11 @@ function customRenderOptions(references) {
         // node contains merged properties of the in-line reference and _references object
         'ImageRef': (node) => {
             // when __typename === ImageRef
-           return <img src={node._path} alt={'in-line reference'} /> 
+            return <img src={node._path} alt={'in-line reference'} />
         },
         'AdventureModel': (node) => {
             // when __typename === AdventureModel
-            return <Link to={`/adventure:${node.slug}`}>{`${node.title}: ${node.price}`}</Link>;
+            return <Link to={`/adventure/:${node.slug}`}>{`${node.title}: ${node.price}`}</Link>;
```

Comment:
```suggestion
            return <Link to={`/adventure/${node.slug}`}>{`${node.title}: ${node.price}`}</Link>;
```

Does this need the `:` in `/adventure/:${node.slug}`